src/consts.ts

```diff
@@ -20,11 +20,7 @@ export enum HubType {
 
 
 // tslint:disable-next-line
-export let HubTypeNames = Object.keys(HubType).reduce((result: {[hubType: string]: string}, item) => {
-    // @ts-ignore
-    result[HubType[item]] = item;
-    return result;
-}, {});
+export const HubTypeNames = HubType;
```

Comment:
TS enums with number values include the reverse mapping already, so the code above produced HubTypeNames which was almost identical in content to HubType enum, but the old `DeviceTypeNames` where causing problems with TS typing (previously they were all unused)